WebDriverAgentTests/IntegrationTests/FBPasteboardTests.m

```diff
@@ -40,8 +40,11 @@ - (void)testSetPasteboard
   [textField tap];
   XCTAssertTrue([textField fb_clearTextWithError:&error]);
   [textField pressForDuration:2.0];
-  XCUIElement *pasteItem = [[self.testedApplication descendantsMatchingType:XCUIElementTypeAny]
-                            matchingIdentifier:@"Paste"].fb_firstMatch;
+  XCUIElementQuery *pastItemsQuery = [[self.testedApplication descendantsMatchingType:XCUIElementTypeAny] matchingIdentifier:@"Paste"];
+  if (![pastItemsQuery.element waitForExistenceWithTimeout:2.0]) {
+    XCTAssertFalse(true, @"No matched element named 'Paste'");
```

Comment:
there is XCTFail or something like this to fail a test immediately

Reply:
https://developer.apple.com/documentation/xctest/failing_unconditionally?language=objc